src/sentry/data_export/utils.py

```diff
@@ -14,6 +15,11 @@
 def snuba_error_handler(logger):
     try:
         yield
+    except discover.InvalidSearchQuery as error:
+        metrics.incr("dataexport.error", tags={"error": six.text_type(error)}, sample_rate=1.0)
+        logger.info("dataexport.error: {}".format(six.text_type(error)))
```

Comment:
Can this error message ever contain unicode data? If so you should put a `u` prefix on the format string.

Reply:
Looks like all of the exceptions here can contain unicode data. Is it safe to just use the `u` prefix for all of them?